pkgs/applications/video/anime-downloader/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, python3, aria2, mpv, nodejs, fetchFromGitHub }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "anime-downloader";
+  version = "5.0.14";
+
+  src = fetchFromGitHub {
+    owner = "anime-dl";
+    repo = "anime-downloader";
+    rev = version;
+    sha256 = "sha256-Uk2mtsSrb8fCD9JCFzvLBzMEB7ViVDrKPSOKy9ALJ6o=";
+  };
+
+  propagatedBuildInputs = [
```

Comment:
Have you tried this with `buildInputs` instead of `propagatedBuildInputs`?

`propagated...` is used in some very restricted and specialized scenarios.

Indeed I would expect that only the Python packages you listed would be in `propagated` while the other 3 would be in `buildInputs`.